tests/suites/deploy_aks/task.sh

```diff
@@ -0,0 +1,30 @@
+test_deploy_aks() {
+	# TODO(anvial): we need to enable this test after we add support of k8s tooling in strict snap
+	# https://github.com/juju/juju/blob/develop/cmd/juju/caas/add.go#L240-L245
+	# shellcheck disable=SC2160
+	if [ true ]; then
+		echo "==> TEST SKIPPED: Deploy aks tests"
+		return
+	fi
+
+	set_verbosity
+
+	echo "==> Checking for dependencies"
+	check_dependencies juju
```

Comment:
We should have added `az` to this, as we need to ensure the dependency existed before running the task.

We can do this in a follow up PR though.